ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml

```diff
@@ -187,7 +189,7 @@ StatusSectionLayout {
             isControlNode: root.isControlNode
             communitySettingsDisabled: root.communitySettingsDisabled
             overviewChartData: rootStore.overviewChartData
-            shardingEnabled: localAppSettings.wakuV2ShardedCommunitiesEnabled ?? false
+            shardingEnabled: !isAdmin && !isTokenMaster && localAppSettings.wakuV2ShardedCommunitiesEnabled
```

Comment:
Similar here; is it possible to be the "owner" but at the same time also the `token master" @jrainville ?

Reply:
In a feature sense, yes, an Owner has all the same rights as a TM, but code wise, we mean an actual TM with `isTokenMaster`. So you're code is correct.
```qml
isTokenMaster: memberRole === Constants.memberRole.tokenMaster
```